app/helpers/buyers/accounts_helper.rb

```diff
@@ -6,7 +6,7 @@ def public_domain(account)
   end
 
   def account_title account
-    [ h(account.org_name), h(account.admin_user.try!(:display_name)) ].compact.join(" &mdash; ").html_safe
+    [ h(account.org_name), h(account.decorate.admin_user_display_name) ].compact.join(" &mdash; ").html_safe
```

Comment:
I think we could use the decorated account in the controller instead of calling decorate in the helpers.